### PR TITLE
Update Supabase Icon

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
**Issue:** https://github.com/simple-icons/simple-icons/issues/7875

**Similarweb rank:** [49,829](https://www.similarweb.com/website/supabase.com/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Update the Supabase icon to match the latest Supabase logo in the Github repo.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10838,7 +10838,7 @@
         {
             "title": "Supabase",
             "hex": "3ECF8E",
-            "source": "https://github.com/supabase/supabase/blob/2a983c3290148d17cfce9e34c0a39102b22fdf78/web/static/img/showcase-logo/supabase-logo.svg"
+            "source": "https://github.com/supabase/supabase/blob/master/packages/common/assets/images/supabase-logo-icon.svg"
```
Comment (LitoMore) [inline _data/simple-icons.json:10841]: Please use the permalink instead.

File: icons/supabase.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Supabase</title><path d="M21.362 9.354H12V.396a.396.396 0 0 0-.716-.233L2.203 12.424l-.401.562a1.04 1.04 0 0 0 .836 1.659H12v8.959a.396.396 0 0 0 .716.233l9.081-12.261.401-.562a1.04 1.04 0 0 0-.836-1.66z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Supabase</title><path d="M11.9925 15.4495s-7.41792-.0197-9.58941.0098c-1.37891.0188-2.121541-1.3022-2.086558-2.0935.034849-.7882.336538-1.1521.56419-1.4376C1.98747 10.5403 9.7708.733045 10.0033.43219c.2783-.3600834.578-.42286158.9104-.42249002.5694.00063652.9785.58089502.9802.96149402.0092 2.095106.0507 7.573926.0507 7.573926s9.3704-.00117 9.8064.01307c1.1568.03778 1.8149 1.11527 1.8865 1.68721.0219.1754.1826.9291-.3642 1.6248-2.5597 3.2567-9.0729 11.4572-9.1807 11.5801-.1616.1844-.3602.5549-1.0044.543-.6785-.0126-.9782-.6694-.9818-.8802-.0355-2.0676-.1139-7.6636-.1139-7.6636Z"/></svg>
```
Comment (LitoMore) [inline icons/supabase.svg:1]: Could you remove the newline at the end of the file?

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: icons/supabase.svg (hunk 2 above)
Comment (Isaiah-Hamilton) [reply icons/supabase.svg:1]: I don't see any newline at the end of the file.

Comment (Isaiah-Hamilton) [reply icons/supabase.svg]: never mind I found it.

**Review by LitoMore — CHANGES_REQUESTED** [review]
CI still fails. See https://github.com/simple-icons/simple-icons/actions/runs/3133553567/jobs/5087053384.

You must ensure your icon size is correct and center-aligned to the view box.

**Comment by Isaiah-Hamilton** [comment]
Sorry for the run around but CI is now passing.

**Review by LitoMore — CHANGES_REQUESTED** [review]
CleanShot 2022-09-27 at 15 13 36@2x

I see some differences. Its aspect ratio should not be changed.

Your aspect ratio:

CleanShot 2022-09-27 at 15 15 06@2x


Expected aspect ratio:

CleanShot 2022-09-27 at 15 14 41@2x

Please make sure the aspect ratio lock is enabled when you resize the icon.

**Comment by LitoMore** [comment]
@Isaiah-Hamilton Could you fix it?

**Comment by Isaiah-Hamilton** [comment]
Sorry for the very late response. Sadly I couldn't fix or reproduce the same bug.

**Comment by LitoMore** [comment]
@Isaiah-Hamilton No worries. Let me help you out.

**Pull request closed without merging**